### DISCUSSION TITLE
Loading a flight plan from littlenavmap

### DISCUSSION THREAD

**Discussion by assafco1000** [flybywiresim/aircraft#7237] [Help]
Hi,

I was using littlenav map (Don't have a simbrief or navigraph account) to create and load flight plans to fbw A320. What I did was saving the pln file and loading it with msfs and then start the flight. After  latest update to V0.8.1 it doesn't load the flight plan and the mdcu is empty (doesn't show any plan or legs).
Am I doing something wrong here?

**Comment by Sankalp1465** [comment]
You Have to use Simbrief for A32NX it doesnt works with MSFS flight plan and BTW if you don't know Simbrief is free I also use that

**Comment by AdamNZ** [comment]
Simbrief is essential, I'm afraid - but it's easy to use (and free).

My procedure is:
1) Create the flight plan in Simbrief and make it active (generate route).
2) If you're using the latest (dev) Littlenavmap, you can import the active Simbrief flight plan (you need to be logged into Simbrief for this to work).
3) With LNM open, start the 320 and import the Simbrief flight plan.
4) Using LNM to show DEP and ARR procedures, set up your route in the MCDU, choosing the most suitable procedures.
5) Copy any additions/changes to the MCDU flight plan back into LNM and save as .lnmpln (for future reference). 

Ideally, I'd love to have the facility to save (then reload) an MCDU flight plan (as a company route):
https://github.com/flybywiresim/a32nx/issues/6178
